packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroupPage.tsx

```diff
@@ -32,8 +32,55 @@ export class ResizeGroupPage extends React.Component<any, any> {
+                of the ResizeGroup provides some initial data, a reduce function, and a render function. The render function is responsible
+                for populating the contents of a the container when given some data. The initial data should represent the data that should
+                be rendered when the ResizeGroup has infinite width. If the contents returned by the render function do not fit within the
+                ResizeGroup, the reduce function is called to get a version of the data whose rendered width should be smaller than the data
+                that was just rendered.
+              </p>
+              <p>
+                An example scenario is shown below, where controls that do not fit on screen are rendered in an overflow menu. The data in
+                this situation is a list of 'primary' controls that are rendered on the top level and a set of overflow controls rendered in
+                the overflow menu. The initial data in this case has all the controls in the primary set. The implementation of onReduceData
+                moves a control from the overflow well into the primary control set.
+              </p>
+              <p>
+                This component queries the DOM for the dimensions of elements. Too many of these dimension queries will negatively affect
+                the performance of the component and could cause poor interactive performance on websites. One way to reduce the number of
+                measurements performed by the component is to provide a cacheKey in the initial data and in the return value of
+                onReduceData. Two data objects with the same cacheKey are assumed to have the same width, resulting in measurements being
+                skipped for that data object. In the controls with an overflow example, the cacheKey is simply the concatenation of the
+                keys of the controls that appear in the top level.
+              </p>
+            </span>
+          </div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>
+                Ensure the width of the parent of this component has a fixed width that does not depend on the dimensions of it's children.
+                Failure to do so may result in ResizeGroup attempting to fill a width of 0px.
+              </li>
+              <li>
+                Ensure that result of rendering the data returned by onReduceData is actually smaller than the previous data.
+              </li>
+              <li>
+                Include a cacheKey in your data to improve performance. Two objects with the same cacheKey are assumed to have the same
+                width, so the ResizeGroup will only store one measurement per resize group.g
+               </li>
+            </ul>
+          </div>
+        }
+        donts={
+          <div>
+            <ul>
+              <li>Do any DOM measurements inside your onReduce function as this will degrade performance</li>
```

Comment:
It's effectively in a the do's with: " Ensure the width of the parent of this component has a fixed width that does not depend on the dimensions of it's children ..." Do you want me to put it in the don'ts as well?